src/pages/studies/studies.tsx

```diff
@@ -11,16 +11,15 @@ import { useFilters } from './utils';
 
 import styles from './studies.module.scss';
 
-const studiesPerPage = 10;
+const MAX_NUMBER_STUDIES = 1000;
 
 const Studies = () => {
   const { filters } = useFilters();
-  const [currentPage, setCurrentPage] = useState(1);
 
   let studiesResults = useGetStudiesPageData({
     sqon: filters,
-    first: studiesPerPage,
-    offset: (currentPage - 1) * studiesPerPage,
+    first: MAX_NUMBER_STUDIES,
```

Comment:
hum i'm not sure but I would switch first and offset : 
first: 0,
offset: MAX_NUMBER_STUDIES

instead of

first: MAX_NUMBER_STUDIES,
offset: 0

Reply:
humm.. no that would not work.